### PR TITLE
Combined fixes

### LINKED ISSUE #1785: [DEFECT] Can't find windows conda defs.
### Thank you for the defect report

- [X] I am using the latest version of `RAVEN`.
- [X] I have read the [Wiki](https://github.com/idaholab/raven/wiki).
- [ ] I have created a [minimum, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)
that demonstrates the defect.

### Defect Description

scripts/establish_conda_env.sh uses the wrong test to check for windows.

### Steps to Reproduce

Run on windows without specifying conda definition location.

### Expected Behavior

Find the default miniconda location.

### Screenshots and Input Files

_No response_

### OS

Windows

### OS Version

_No response_

### Dependency Manager

CONDA

### For Change Control Board: Issue Review

- [x] Is it tagged with a type: defect or task?
- [x] Is it tagged with a priority: critical, normal or minor?
- [x] If it will impact requirements or requirements tests, is it tagged with requirements?
- [x] If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [x] Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

### For Change Control Board: Issue Closure

- [x] If the issue is a defect, is the defect fixed?
- [x] If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by joshua-cogliati-inl**
--------
Pull Request Description
--------
##### What issue does this change request address?
Closes #1785 

##### What are the significant changes in functionality due to this change request?
Updates ray.
Fixes test to find windows conda defs.
Also removes Prescient warning when not needed.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?
- [x] 9. If any test used as a basis for documentation examples (currently found in `raven/tests/framework/user_guide` and `raven/docs/workshop`) have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

File: dependencies.xml [hunk 1]
```diff
@@ -16,6 +16,8 @@ Note all install methods after "main" take
  add library from forge:     <library source='forge'>...
  optional:                   <library optional='True'>...
  skip run/install check:     <library skip_check='True'>...
+ add pip library with extra parameters:
+                             <ray source="pip" pip_extra="[tune]">1.9</ray>
```
Comment (wangcj05) [inline dependencies.xml:20]: Could you add a little bit more descriptions about this feature? Maybe add an example
```
pip install -U "ray[tune]"  # installs Ray + dependencies for Ray Tune
pip install -U "ray[rllib]"  # installs Ray + dependencies for Ray RLlib
pip install -U "ray[serve]"  # installs Ray + dependencies for Ray Serve
```

**Review by wangcj05 — CHANGES_REQUESTED** [review]
@joshua-cogliati-inl I have one minor comment for your consideration.

File: dependencies.xml (hunk 1 above)
Comment (joshua-cogliati-inl) [reply dependencies.xml:20]: Good point. I expanded the example.

**Review by wangcj05 — APPROVED** [review]

File: framework/CodeInterfaces/Prescient/PrescientCodeInterface.py [hunk 2]
```diff
@@ -26,8 +26,8 @@
   prescient = pkg_resources.get_distribution("prescient")
   prescientLocation = prescient.location
 except Exception as inst:
-  warnings.warn(f"Finding Prescient failed with {inst}")
   prescientLocation = None
+  prestientException = inst
```
Comment (mandd) [inline framework/CodeInterfaces/Prescient/PrescientCodeInterface.py:30]: is there a typo here? prescientException instead of prestientException

Comment (joshua-cogliati-inl) [reply framework/CodeInterfaces/Prescient/PrescientCodeInterface.py]: Good point.  Fixed.

**Comment by joshua-cogliati-inl** [comment]
@mandd @wangcj05 Changes made, and the tests have passed.

**Review by wangcj05 — APPROVED** [review]

**Comment by wangcj05** [comment]
checklist is good, PR is good to be merged.

**Pull request merged**